CMake/ctest_script.cmake

```diff
@@ -238,16 +238,25 @@ IF ( QMC_COMPLEX )
   SET( CTEST_OPTIONS "${CTEST_OPTIONS};-DQMC_COMPLEX='${QMC_COMPLEX}'" )
 ENDIF()
 
+#Always explicitly enable/disable MPI in testing
 IF ( DEFINED QMC_MPI )
   SET( CTEST_OPTIONS "${CTEST_OPTIONS};-DQMC_MPI='${QMC_MPI}'" )
+ELSE()
+  SET( CTEST_OPTIONS "${CTEST_OPTIONS};-DQMC_MPI=0" )
```

Comment:
This breaks other nightly testing scripts without specifying QMC_MPI and this is the opposite of our default.

Reply:
ah yes, we need "if (defined var ) {} else { set default }" assuming a value =0 counts as defined in cmake speak

Reply:
Fix pushed